### ISSUE TITLE
Request to update source distributions in pypi repository

### ISSUE THREAD

**Issue by pvsmpraveen** [tensorflow/data-validation#235]
Hello Developers/Code Maintainers,

Our project has dependency to import the sources from the pypi repository, transform and build a docker image from it.

Our build-system is breaking because this package does not have Source Distributions in https://pypi.org/project/tensorflow-data-validation/1.11.0/#files

We request to publish the sources to pypi repository in the files section.

Thank you!

**Comment by gaikwadrahul8** [comment]
Hi, @pvsmpraveen 

Apologize for the delayed response and I checked previous releases of `tensorflow-data-validation` there also No source distribution files available and See tutorial on [generating distribution archives](https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives) so are you looking to add distribution files [here](https://pypi.org/project/tensorflow-data-validation/1.1.0/#files) because at the moment No source distribution files available ? I'm not sure whether we can add it from our end or not at the moment. Thank you!

**Comment by pvsmpraveen** [comment]
Yes, i am looking to add distribution files there.. all our other dependencies like [Numpy](https://pypi.org/project/numpy/#files), [Keras](https://pypi.org/project/keras/#files), [Pandas](https://pypi.org/project/pandas/#files), [Pyarrow](https://pypi.org/project/pyarrow/#files) have these files except this one package.. 

Since you are the owner of this package, you would have permission to upload it?

**Comment by pvsmpraveen** [comment]
Any update?
